app/upgrades/v8/upgrades.go

```diff
@@ -16,6 +18,15 @@ func CreateUpgradeHandler(
 	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
 		ctx.Logger().Info("start to run module migrations...")
 
+		// Add atom name and symbol into the bank keeper
+		atomMetaData, found := keepers.BankKeeper.GetDenomMetaData(ctx, "uatom")
+		if !found {
+			return nil, errors.New("atom denom not found")
+		}
+		atomMetaData.Name = "Cosmos Hub Atom"
```

Comment:
```suggestion
		atomMetaData.Name = "Cosmos"
```

Reply:
I feel like we should change the registry

Reply:
@okwme https://github.com/cosmos/chain-registry/pull/848 is pending, if we merge that then we could merge this as is

Reply:
:+1: